frontend/packages/kubevirt-plugin/src/k8s/wrapper/vm/combined-disk.ts

```diff
@@ -343,7 +340,7 @@ export class CombinedDiskFactory {
               (getOwnerReferences(p) || []).some((ownerReference) =>
                 compareOwnerReference(ownerReference, {
                   name: dataVolumeName,
-                  kind: kubevirtReferenceForModel(DataVolumeModel),
+                  kind: DataVolumeModel,
```

Comment:
Shouldn't this be...?

```suggestion
                  kind: DataVolumeModel.kind,
```

Reply:
oops